content/docs/preferences.md

```diff
@@ -313,6 +314,7 @@ A variety of settings that didn't fit elsewhere
     * `☐ Update silently`: instead of prompting, will automatically reload the file from disk
     * `☐ Scroll to the last line after update`: will scroll to the end of the file after reloading from disk (otherwise, the cursor and scrolled-location stays where it was before the update)
 * `☐ Enable Notepad++ auto-updater`: will automatically download updates from the official website, once the development team has decided it's time to push an update to users.  If disabled, you will have to manually download the installer from the official website yourself.
+* `☐ Set Save dialog file extension filter... for Normal Text`: when this is enabled and the current file is set to Normal Text, the **Save As** dialog's filter will default to `*.*` and require you to type an extension; when disabled, the filter will default to `*.txt` and will auto-append the `.txt` extension if you don't type an extension or choose a different filter.  (Regardless of the state of this option, you can always select a different filter manually in the **Save As** dialog.)
```

Comment:
"and the current file is set to Normal Text" ---> "and the current file's language is set to Normal Text" ??

Somehow make a note that this is only for "untitled files", you know, "new 2" and the like.  (If you do "Save As" on an existing "test.txt", it won't set the filter to \*.\*)

Reply:
clarified